src/move.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
Yes, they do not need to be objects. But is there any reason they can't be objects?  I find that having them as objects makes the code more testable.